todo/src/routes/routes.ts

```diff
@@ -0,0 +1,133 @@
+import { server } from "../server"
+import { ResponseToolkit, Request } from "@hapi/hapi"
+import { tasks } from "../models";
+
+export function getRoute() {
```

Comment:
So we are exporting a function that sets the routes? I am unclear why this is wrapped in a function when you could just export the routes as an array